<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="MyWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>761</width>
       <height>391</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_onlinenum">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>FA01XX(XX:32-c8,值越低发现物体概率越高) FA02XX(XX:32-c8,值越低发现物体概率越高) FA07XX(XX:02代表检测0.2米/s以上速度)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_max_angle">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="sentToClientEditLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>发送命令</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>开启服务器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>清空信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearPoints">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>清除点痕</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>关闭服务器</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QWidget</extends>
   <header location="global">mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
